cluster/swarm/cluster.go

```diff
@@ -186,6 +186,22 @@ func (c *Cluster) Image(IdOrName string) *cluster.Image {
 	return nil
 }
 
+func (c *Cluster) Pull(name string, callback func(string, string)) {
+	size := len(c.nodes)
+	done := make(chan bool, size)
+	for _, n := range c.nodes {
+		go func(nn *node) {
+			callback(nn.Name(), "")
```

Comment:
I will even though I don't think is is necessary. The function isn't called by the cluster driver implem, but by the swarm api. So we won't call it with nil.